scripts/setup-macos.sh

```diff
@@ -109,6 +109,16 @@ function install_re2 {
   cmake_install -DRE2_BUILD_TESTING=OFF
 }
 
+function install_protobuf {
+  # Protobuf's optimized build and debug build are incompatible when linked with code from
```

Comment:
I'm curious what causes Velox build to list against both build and debug versions of the protobuf. CC: @rui-mo